### PR TITLE
Completable level thumbnail component

### PR DESCRIPTION
**Pull request by joshlory**
Create a green checkmark overlay React component for puzzle thumbnails.

screen shot 2017-07-20 at 3 12 20 pm

+cc @ryansloan

### CODE REVIEW

**Comment by ryansloan** [comment]
Terrific! And we can reuse it elsewhere if we want 💯  

It's a bit hard to differentiate between the pass and perfect conditions but shippable. Let me know when it's in a state when I can go and just hack it in the inspector to find something that looks a bit more distinctive.

File: apps/src/code-studio/components/stageExtras/CompletableLevelThumbnail.story.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+export default storybook => {
+  storybook
+    .storiesOf('CompletableLevelThumbnail', module)
+    .addWithInfo(
+      'Overview',
+      'This is the CompletableLevelThumbnail component.',
+      () => (
+        <div>
+          <CompletableLevelThumbnail size={200} completed={true} style={{float: 'left'}}>
```
Comment (Hamms) [inline apps/src/code-studio/components/stageExtras/CompletableLevelThumbnail.story.jsx]: is the float here for a reason?

**Comment by joshlory** [comment]
@ryansloan actually these are both the same state — the backgrounds make the tint look different.  My plan was to only show the checkmark if they perfect the bonus level.  Should I make it work differently?

**Comment by ryansloan** [comment]
@joshlory Got it! What you built is what's spec'd, which is just perfecting, I just thought you had jumped the gun. :shipit: for now and we may want to consider an alternative display for passing after that.

**Pull request merged**